src/events/messageCreate.ts

```diff
@@ -0,0 +1,21 @@
+import { Client, Message, MessageFlags, TextChannel } from 'discord.js';
+import { deleteAdvertisingMessage } from '../utils/helpers';
+import config from "../config/config";
+
+export const onMessageCreate = async (client: Client, message: Message) => {
+    console.log(message.channel.id)
+    if (message.channel.id !== config.advertisingId) return;
+    if (message.author.bot) return;
+
+    const channel = message.channel as TextChannel;
+    await deleteAdvertisingMessage(client);
+
+    try {
+        await channel.send({
+            content: config.advertisingMessage,
+            flags: [ MessageFlags.SuppressNotifications ] // @silent message
+        });
```

Comment:
Can you send an ephemeral message here or is that interaction-only?

Reply:
![grafik](https://github.com/user-attachments/assets/0adf342b-30ad-4a3e-b9c6-8f8f59306621)
Seems like no